### PR TITLE
Convert image to vector SVG

### PR DESCRIPTION
**Pull request by thisisharrsh**
Fixes : #45963

### CODE REVIEW

**Review by sftim — DISMISSED** [review]
Thanks, but that's a raster image @thisisharrsh 

You need to redraw the image as SVG (possibly using the Gimp `.xcf` source file for hints)

**Comment by thisisharrsh** [comment]
> You need to redraw the image as SVG

Hi @sftim, I have redrawn the image. But in the preview, there is a font mismatch.

**Comment by sftim** [comment]
We recommend converting text to curves @thisisharssh; you could try that.

Don't worry if the new image isn't a pixel accurate match. It's important to convey the same information and for the images to look similar.

**Review by sftim — COMMENTED** [review]
There's a lot of whitespace around the image; otherwise, it looks OK. Thanks for the work so far.

**Comment by thisisharrsh** [comment]
Hi @sftim, I'll remove the whitespace from the image.

**Comment by thisisharrsh** [comment]
> There's a lot of whitespace around the image

Hi @sftim, i have removed the extra whitespace from the image.

**Comment by thisisharrsh** [comment]
Please take a look.

**Comment by nate-double-u** [comment]
Hi @thisisharrsh, this image renders very small (tested on Firefox & safari), and so is not legible. Are you able to make it bigger?

Screenshot 2024-07-04 at 11 24 35 AM
Screenshot 2024-07-04 at 11 26 34 AM

File: static/images/blog/2024-04-22-userns-beta/image.svg
Comment (sftim) [inline static/images/blog/2024-04-22-userns-beta/image.svg:1]: Try this:
```diff
-<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" id="svg5" version="1.1" viewBox="27.88135593220339 2.97 163.72881355932205 95.04" style="max-height: 500px" width="163.72881355932205" height="95.04">
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" id="svg5" version="1.1" viewBox="27.88135593220339 2.97 163.72881355932205 95.04">
```

File: content/en/blog/_posts/2024-04-22-userns-beta/index.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ signals to, etc. Furthermore, capabilities granted are only valid inside the
 user namespace and not on the host, limiting the impact a container
 escape can have.
 
-{{< figure src="/images/blog/2024-04-22-userns-beta/userns-ids.png" alt="Image showing IDs 0-65535 are reserved to the host, pods use higher IDs" title="User namespace IDs allocation" >}}
+{{< figure src="/images/blog/2024-04-22-userns-beta/image.svg" alt="Image showing IDs 0-65535 are reserved to the host, pods use higher IDs" title="User namespace IDs allocation" >}}
```
Comment (sftim) [inline content/en/blog/_posts/2024-04-22-userns-beta/index.md:65]: Try this:
```suggestion
{{< figure src="/images/blog/2024-04-22-userns-beta/image.svg" alt="Image showing IDs 0-65535 are reserved to the host, pods use higher IDs" title="User namespace IDs allocation" class="diagram-medium" >}}
```

**Comment by thisisharrsh** [comment]
Hi, @nate-double-u   @sftim  made the suggested changes and attached the screen shot for a quick look at the changes made. 

image

**Comment by thisisharrsh** [comment]
_Deployment preview: https://deploy-preview-45968--kubernetes-io-main-staging.netlify.app/blog/2024/04/22/userns-beta/#what-is-a-user-namespace_

**Review by nate-double-u — COMMENTED** [review]
Thanks for this @thisisharrsh!

/approve

**Pull request merged**